finalPrep/README.md

```diff
@@ -23,51 +23,51 @@ Some things to consider as you refactor include...
-- [ ] Study/Project work time (usually about 25 hours/week outside of class time)
-- [ ] Personal, Health + Wellness
+- [x] Standard Mod 1 class schedule (find the Mod 1 calendar [on the Front End curriculum site](https://frontend.turing.edu/))
+- [x] Study/Project work time (usually about 25 hours/week outside of class time)
+- [x] Personal, Health + Wellness
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+- <img width="1440" alt="Screen Shot 2022-01-15 at 12 53 13 PM" src="https://user-images.githubusercontent.com/92230099/149637708-b5793378-b36a-480f-acbf-42cf3cf29128.png">
+- <img width="1440" alt="Screen Shot 2022-01-15 at 12 53 20 PM" src="https://user-images.githubusercontent.com/92230099/149637732-7e1243dd-00ef-4601-8932-47235d8d70bf.png">
+- <img width="1440" alt="Screen Shot 2022-01-15 at 12 53 23 PM" src="https://user-images.githubusercontent.com/92230099/149637741-c6937a6f-9727-49c3-9357-9d5fb59c08c7.png">
 
 ## Mentorship Prep
 
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
 
-- [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+- [x] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
+  - [x] Add link to your gist here: [Mentorship Prep](https://gist.github.com/DanielN88/205f236e69e050edba1731b78f1dbadc)
 
 ## Lesson Prep
 
 You've learned a lot about how to take strong notes during Mod 0. Show us your skills while you learn how to pre-teach content for your first lesson in Mod 1!
 
-- [ ] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
-  - [ ] Add a link to your gist here:
+- [x] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
+  - [x] Add a link to your gist here: [Pre Teaching Prep](https://gist.github.com/DanielN88/cdcfceb0969388657615492703e443f0)
 
 ## Group Work Prep
 
 As part of Turing's project-based learning approach, you will often be working in pairs or larger groups. In order to set yourself (and your team) up for success, it is important to ensure you are prepared to be an equitable contributor and teammate.
 
-- [ ] Complete the [DTR Guiding Questions](https://gist.github.com/ericweissman/c56f3a98cdce761808c21d498a52f5c6)
-  - [ ] Add a link to your gist here: 
+- [x] Complete the [DTR Guiding Questions](https://gist.github.com/ericweissman/c56f3a98cdce761808c21d498a52f5c6)
+  - [x] Add a link to your gist here: [Group Work Prep](https://gist.github.com/DanielN88/ab5fa38211a4e091c33326cfa5416fc9)
```

Comment:
I appreciate your attention to detail on the group work pre-DTR. Save this document because you'll be doing a DTR every time a new project kicks off, and since you've put all the work in now you will save yourself a lot of time and effort of rehashing everything here.